hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -800,14 +800,14 @@ private MatchedIndex collectMatchedIndex(SchemaLabel schemaLabel,
         Set<IndexLabel> ils = InsertionOrderUtil.newSet();
         for (Id il : schemaLabel.indexLabels()) {
             IndexLabel indexLabel = schema.getIndexLabel(il);
-            if (indexLabel.indexType().isUnique()) {
+            if (indexLabel == null || indexLabel.indexType().isUnique()) {
```

Comment:
add some comment like:
schema.getIndexLabel() may return null here because the IndexLabel is being created at this time and has not been saved to the backend storage